queries/query/highlights.scm

```diff
@@ -19,5 +19,6 @@
   ")"
 ] @punctuation.bracket
 
-((program . (comment) @include)
- (#match? @include "^;\ +inherits\ *:"))
+;; TODO: fix this query
+;((program . (comment) @include)
```

Comment:
looks like the error here is because of the `program` node. Looks like it breaks when it has a child, when capturing the program node by itself it doesn't error `(program) @foo`

Reply:
The query also fails in the ts-cli web-ui, so maybe a bug in ts.

Reply:
Looks like something to do with having comment in  extras.

This doesn't work

```
(program (comment) @foo)
```

This works

```
(program (named_node) @foo)
```

Reply:
https://github.com/tree-sitter/tree-sitter/issues/967

Reply:
This was fixed upstream, I guess we need to re-generate the grammar.